### PR TITLE
Test for Subtraction done

### PR DESCRIPTION
**Pull request by d-trim**

### CODE REVIEW

File: app/src/test/java/com/example/mediaplayer1/ExampleUnitTest.kt [hunk 1]
```diff
@@ -14,4 +14,8 @@ class ExampleUnitTest {
     fun addition_isCorrect() {
         assertEquals(4, 2 + 2)
     }
+    @Test
+    fun subtract(){
```
Comment (shankhadeep-ghoshal) [inline app/src/test/java/com/example/mediaplayer1/ExampleUnitTest.kt]: Please rename it to `subtract_isCorrect` keeping up with the naming convention

**Review by shankhadeep-ghoshal — CHANGES_REQUESTED** [review]
Changes requested

File: app/src/test/java/com/example/mediaplayer1/ExampleUnitTest.kt (hunk 1 above)
Comment (d-trim) [reply app/src/test/java/com/example/mediaplayer1/ExampleUnitTest.kt]: done

**Review by shankhadeep-ghoshal — APPROVED** [review]
LGTM!

**Pull request merged**